frontend/demo/src/pages/Inbox/NoConversations/index.module.scss

```diff
@@ -0,0 +1,12 @@
+.component {
+  margin: 1.5em;
+}
+
+.component strong {
```

Comment:
I think it would read nicer if both `strong` and `p` would be nested in `.component`